aws-java-workshop/src/main/java/com/emc/vipr/s3/sample/_06_ReadObjectWithMetadata.java

```diff
@@ -15,6 +15,9 @@
 package com.emc.vipr.s3.sample;
 
 import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3Client;
```

Comment:
It actually looks like you added AmazonS3Client in all the classes, but I don't think we need this anymore.